src/bitcoin-cli.cpp

```diff
@@ -85,7 +85,7 @@ static bool AppInitRPC(int argc, char* argv[])
         }
 
         fprintf(stdout, "%s", strUsage.c_str());
-        return false;
+        if(argc < 2) return false; else exit(0);
```

Comment:
Ok, that makes sense. How about simply returning `int` instead of `bool` from `AppInit*`?
And then
```
        int ret = AppInitRPC(argc, argv);
        if (ret != -1)
            return ret;
```
This option seems a bit cleaner to me and should do the trick, see 0f9d7b6.